.eslintrc.json

```diff
@@ -76,7 +75,7 @@
         "no-alert": 2,
         "no-array-constructor": 2,
         "no-caller": 2,
-        "no-case-declarations": 2,
+        "no-case-declarations": 0,
```

Comment:
Can you change this back? Declarations are still allowed in case statements when you surround the block in `{}` like:
```javascript
switch(foo) {
case 'bar': {
    let a = 3;
    return a;
}
}
```
That way, it'll still prevent declarations from one case statement from bleeding over into others.